mautic/mautic
# Transliterate filenames in elFinder

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

It's better sanitize filenames and directories in elFinder to prevent use special characters, accents and empty spaces. This PR deliver it



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  Try create directory  `directory with čšťĺé`
3. elFinder should create `directory-with-cstle`
4. Try upload file `filename with čšťĺé.png`
5. elFinder should upload `filename-with-cstle.png`




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11324"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
